code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -54,16 +50,14 @@
 	if(!.)
 		return FALSE
 
-	clear_alerted()
+	LAZYINITLIST(alerted)
 	var/do_alert = (COOLDOWN_FINISHED(src, alert_cooldown) && (locate(/mob/living) in contents))
 	if(!do_alert)
 		return TRUE
 
 	// Cache the list before we open the box.
-	alerted = list()
```

Comment:
You have to still clear the list before filling it. LAZYLISTINIT macro won't do that for you. 

Put alerted.Cut() here

Reply:
Your right hmmm. You can never to be too cautious so why not